docs/attribution/0001-analytics.rst

```diff
@@ -339,6 +364,97 @@ There are two ways to use TrackProductDownload:
 You do NOT need to include ``datalayer-productdownload-init.es6.js`` in the page bundle, it is already included
 in the site bundle.
 
+Widget Action
+~~~~~~~~~~~~~
+
+We are using the custom event ``widget_action`` to track the behaviour of javascript widgets.
+
+
+.. list-table:: How do you chose between ``widget_action`` and ``cta_click``?
+   :widths: 50 50
+   :header-rows: 1
+
+   * - widget_action
+     - cta_click
+   * - The action is specific or unique.
+
+       *(Only the language switcher changes the page language.)*
+     - The action is "click".
+   * - The user does not leave the page.
+     - It sends the user somewhere else.
+   * - It requires Javascript to work.
+     - No JS required.
+   * - It can perform several actions.
+
+       *(A modal can be opened and closed.)*
+     - It does one action.
+   * - There could be several on the page doing different things.
+
+       *(An accordion list of FAQs)*
+     - There could be several on the page doing the same thing.
+
+       *(A download button in the header and footer.)*
```

Comment:
Not something that needs changing necessarily, but I find the source order of this table pretty hard to follow?